client/js/actions/view-action-creator.js

```diff
@@ -11,12 +11,12 @@ var ViewActionCreator = {
     ApiUtils.addEvent(event, ServerActionCreator.eventAdded);
   },
 
-  getDayData: function(){
+  getDayData: function(date){
```

Comment:
at some point it may be good to abstract them into one function. i think that if we were to do that now it would be an early optimization since we haven't solidified the api. this is something we should definitely keep in mind. thoughts?